404.html

```diff
@@ -14,26 +18,39 @@
   <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
-  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
+  <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link href="images/webclip.png" rel="apple-touch-icon">
 </head>
 <body class="body">
-  <div class="navbar">
-    <div class="navbarleft"><img src="images/menu.png" loading="eager" width="25" data-w-id="cd12c241-3f1a-3a66-57e7-386a8708ac86" alt="Menu icon
-" srcset="images/menu-p-500.png 500w, images/menu.png 512w" sizes="(max-width: 767px) 25px, 100vw" class="menu-icon">
-      <a href="index.html" class="link-block tugende-logo w-inline-block">
-        <div class="navlogo">Tugende</div>
-      </a>
-      <a href="safety.html" class="link-block w-inline-block">
-        <div class="navoption">Safety</div>
-      </a>
-      <a href="faq.html" class="link-block w-inline-block">
-        <div class="navoption">FAQ</div>
-      </a>
-      <a href="about-us.html" class="link-block w-inline-block">
-        <div class="navoption">About Us</div>
-      </a>
+  <div data-animation="default" data-collapse="small" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="navbar-3 w-nav">
+    <div class="navbar">
+      <div class="nav-div left">
+        <a href="index.html" class="brand-2 w-nav-brand">
+          <div class="div-block-tugende-logo"><img src="images/tugende-logo-black.png" loading="lazy" sizes="50px" srcset="images/tugende-logo-black-p-500.png 500w, images/tugende-logo-black.png 899w" alt="" class="image-tugende-logo">
+            <h1 class="text-heading navbar-logo">Tugende</h1>
+          </div>
+        </a>
+        <nav role="navigation" class="nav-menu-3 w-nav-menu">
+          <a href="safety.html" class="navbar-links w-nav-link">Safety</a>
+          <a href="faq.html" class="navbar-links w-nav-link">FAQ</a>
+          <a href="about-us.html" class="navbar-links w-nav-link">About Us</a>
+          <a href="sender-signup.html" class="navbar-links mobile-only w-nav-link">Ship Now</a>
+          <a href="courier-signup.html" class="navbar-links mobile-only w-nav-link">Start Earning</a>
+        </nav>
+      </div>
+      <div class="nav-div right">
+        <div class="navbuttons">
+          <a href="sender-signup.html" class="navbutton button buttonhollow w-button">Ship Now</a>
+          <link rel="prefetch" href="/sender-signup">
+          <a href="courier-signup.html" class="navbutton button w-button">Start Earning</a>
+          <link rel="prefetch" href="/courier-signup">
+        </div>
+        <div class="menu-button-2 w-nav-button">
+          <div class="w-icon-nav-menu"></div>
+        </div>
+      </div>
     </div>
+<<<<<<< Updated upstream
```

Comment:
This shouldn't be here. Its used to compare diffs either between branches, stash, or view merge conflicts